app/controllers/shared/api.php

```diff
@@ -92,10 +92,17 @@
 
 $usageDatabaseListener = function (string $event, Document $document, StatsUsage $queueForStatsUsage) {
     $value = 1;
+
     if ($event === Database::EVENT_DOCUMENT_DELETE) {
         $value = -1;
     }
 
+    if ($event === Database::EVENT_DOCUMENTS_DELETE) {
+        $value = -1 * $document->getAttribute('modified', 0);
```

Comment:
What is the modified attribute?

Reply:
The database events for bulk operations fire a document with 2 attributes:

- `$collection`: The collection that the document were created/updated/deleted in
- `modified`: How many documents were create/updated/deleted

Reply:
This isn't user facing, it's the model of the event we fire from the database library, only gets used for usage. User having this attribute will have no effect